tests/foreach/foreach.t

```diff
@@ -0,0 +1,90 @@
+# Setup few testing env variables
+  $ CRAMPTESTDIR=$PWD
+  $ REPOS=$CRAMPTESTDIR/repos
+
+# Copy all the test directories
+  $ mkdir repos && cd repos && cp -r $TESTDIR/cram_dir/test_dir_* ./
+
+# Edit all urls in subprojects
+  $ cd test_dir_1 &&
+  > sed -i -e "s|PH_TESTDIR2|$REPOS/test_dir_2|g" subprojects.quark &&
+  > sed -i -e "s|PH_TESTDIR3|$REPOS/test_dir_3|g" subprojects.quark &&
+  > cd ..
+  $ cd test_dir_2 &&
+  > sed -i -e "s|PH_TESTDIR4|$REPOS/test_dir_4|g" subprojects.quark &&
+  > cd ..
+
+# Setup the fake git repos
+  $ cd test_dir_1 && git init > /dev/null && git add * && git commit -m "1st commit" > /dev/null && cd ..
+  $ cd test_dir_2 && git init > /dev/null && git add * && git commit -m "1st commit" > /dev/null && cd ..
+  $ cd test_dir_3 && git init > /dev/null && git add * && git commit -m "1st commit" > /dev/null && cd ..
+  $ cd test_dir_4 && git init > /dev/null && git add * && git commit -m "1st commit" > /dev/null && cd ..
+
+# Clone the root testing git repo
+  $ cd $CRAMPTESTDIR
+  $ mkdir checkout && cd checkout
+  $ git clone file://$REPOS/test_dir_1 > /dev/null
+  Cloning into 'test_dir_1'...
+
+# Quark UP from the root and setup a new variable ROOT_TEST
+  $ cd test_dir_1
+  $ ROOT_TEST=$CRAMPTESTDIR/checkout/test_dir_1
+  $ quark up >/dev/null 2>&1
```

Comment:
Here you are using the quark that happens to be in `PATH` for the test, while you should be using the one from the current sandbox. Indeed on my machine everything breaks in the moment you invoke `quark foreach` (as the quark in the PATH is the official master, not your branch).

Reply:
Changed in https://github.com/comelz/quark/pull/27/commits/49ad450a7a7dcf202d192e713c949774adbe82da